### ISSUE TITLE
last_video and last_thumbnail are no longer updated

### ISSUE THREAD

**Issue by allenrong88** [twrecked/hass-aarlo#271]
The last working version was 0.111.4, upgraded Hass.io to 0.112.0 last night, stopped working now. Nothing else has changed from my side.

Both params last_video and last_thumbnail are not getting updated even video/ motion is triggered. It will get them updated after rebooting the whole Hass.io then stopped working again.

Thank you Steve.

**Comment by allenrong88** [comment]
Just restored back to 0.111.4, having the exact same issue now. Tried adding host: https://myapi.arlo.com into my config, did not help unfortunately.

**Comment by twrecked** [comment]
Which version of Aarlo are you using? Arlo seemed to have changed something recently and found a bug in my code.

**Comment by allenrong88** [comment]
Should be the latest release, which is v0.6.19.1. Thx

**Comment by twrecked** [comment]
it's fixed in the 0.7 release which I'll be taking out of alpha pretty soon. If you're fine trying a pre-release give it a go.

Basically, Arlo is running slower and it exposed a bug in my code. 0.7 has a much better state machine than 0.6 so getting the fix in was easier.

**Comment by allenrong88** [comment]
Yep sure. I will give it a shot to see how it goes. Thx Steve.

**Comment by twrecked** [comment]
Let me get alpha.5 merged over, it fixes the issue and allows you to set custom retry values for updating the media.

**Comment by allenrong88** [comment]
No prob. I assume once you merge it over, I will be able to get it updated via HACS instead of doing it manually? Thx

**Comment by twrecked** [comment]
Yes. Just tell HACS you want to see pre-releases.

**Comment by allenrong88** [comment]
Stevem would you please let me know how to configure HACS with beta/ pre-release enabled? Tried searching it online, but no luck yet. Thx

**Comment by twrecked** [comment]
The button has gone... I created alpha.6 without the pre-release tag so you should be able to see it.

**Comment by allenrong88** [comment]
Thanks Steve. I will give it a shot.

**Comment by sebr** [comment]
If you select "Reinstall" you get the beta checkbox that you can toggle to enable preview releases. :)

<img width="863" alt="Screen Shot 2020-07-10 at 10 49 09 am" src="https://user-images.githubusercontent.com/81972/87104359-fdfa1a80-c29a-11ea-8844-d04b04c3ebb5.png">

**Comment by twrecked** [comment]
@sebr Thanks! I was going nuts trying to find it...

**Comment by allenrong88** [comment]
Hi Steve, have been using the latest version for a while now and found out it is not as reliable as before. Sometime it works perfectly but sometimes URL for video and thumbnail not getting updated at all. The only way to get it back to work is to get Hass.io rebooted.

**Comment by CrouchingWorm** [comment]
Hi there, I note this is now closed?
I'm seeing my snapshots in HA are not updating when there's motion detected.
The arlo app shows the updates, but not HA, not until I reboot HA. they then update.
I'm running v0.7.0-beta.3, not sure when this first started happening.
I'm piping into deepstack, as a result that hasn't been firing, until I reboot.

**Comment by sebr** [comment]
I'm seeing the same issues with `0.7.0-beta.3`!

**Comment by CrouchingWorm** [comment]
> I'm seeing the same issues with `0.7.0-beta.3`!

Phew! I’m glad it’s not just me.
I tried monitoring for events and wasn’t able to see any firing after motion was detected. But was able to manually fire them. 
I guess we sit tight and monitor this thread to see what we can to help work this out,

**Comment by twrecked** [comment]
This is still open, I closed another related bug. I think this is Arlo buggering about again. Can you try the following in your config and I'll test it out here.

```yaml
aarlo:
  ..your usual config..
  media_retry:
  - 5
  - 15
  - 25
```

**Comment by CrouchingWorm** [comment]
Thanks so much.
I've applied, I'll monitor throughout the day.
Will advise.

**Comment by CrouchingWorm** [comment]
I've checked the online thumbnails from each of my camera's and they're being updated (on AWS).
Including the camera's without the subscription attached.

**Comment by twrecked** [comment]
You shouldn't need a subscription for this to work. What normally happens is Arlo tweaks the backend and web page and I need to modify my code to keep up.

Can you try beta.4, it has better media library parsing.

**Comment by CrouchingWorm** [comment]
I’ve applied the latest beta and it looks fantastic.
Unfortunately, I’ve also applied a subscription service in the mean time,
so I’ve possibly confused things as two things changed simultaneously.

You're fabulous!

**Comment by twrecked** [comment]
Closing, the latest code has a lot of updates in this area. Create a new issue if needed.
